src/main/java/au/org/emii/geoserver/extensions/filters/layer/data/ValueEncoder.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2014 IMOS
+ *
+ * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
+ *
+ */
+
+package au.org.emii.geoserver.extensions.filters.layer.data;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Set;
+import java.util.TimeZone;
+
+public class ValueEncoder {
```

Comment:
I'm going to leave this as is for now, as IMO 'encode' is more descriptive than 'serialise' in this context.